low-level/files/opt/seagate/sspl/conf/support.yaml

```diff
@@ -1,45 +1,17 @@
-# CORTX Python common library.
 # Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
 #
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as published
-# by the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU Affero General Public License for more details.
-# You should have received a copy of the GNU Affero General Public License
-# along with this program. If not, see <https://www.gnu.org/licenses/>.
-# For any questions about this software or licensing,
-# please email opensource@seagate.com or cortx-questions@seagate.com.
-
-# support framework format
-# bundle:
-#    create:
-#      # url is the command one need to invoke. This can be shell script, python script, REST interface.
-#      # following arguments will be passed in.
-#      # args: --id $bundle_id --path $path_to_store --duration $duration --core $flag --modules $list_of_modules    
-#      url: $create_interface_url 
-#
-#    status:
-#      # following arguments will be passed in.
-#      # args: --id $bundle_id 
-#      url: $status_interface_url 
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
 #
-#    delete:
-#      # following arguments will be passed in.
-#      # args: --id $bundle_id 
-#      url: $delete_interface_url 
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
 #
-#    cancel:
-#      # following arguments will be passed in.
-#      # args: --id $bundle_id 
-#      url: $delete_interface_url 
-
-bundle: 
-    create:
-      url: /opt/seagate/cortx/sspl/bin/sspl_support_bundle create
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
 
-    status:
-      url: /opt/seagate/cortx/sspl/bin/sspl_support_bundle status
+support_bundle:
+    - /opt/seagate/cortx/sspl/bin/sspl_support_bundle create
```

Comment:
what about 'status' ? create and get status supported right ?

Reply:
Yes, @cdeshmukh , As discussed and that was decided. but get staus is still not exposed by every component for use by support bundle framework @vastradparayya please confirm. @vastradparayya have created this ticket to deleted/modify existing support.yaml file and update with latest SB Framework confluence page. 
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/316735509/Support+Bundle+Framework#Interim-format-for-support.yaml-file
And as I got some points from @rahul-tripathi-git . I have pasted it here for information.
> [8/26 5:35 PM] Rahul Tripathi
    Hi Satish,
Whatever we discussed holds true. As the design architecture of cortx may/will change, we have put a halt on any such (New) requirements.
You can leave the code for get_status as is for now.

> ​[8/26 11:15 PM] Satish Darade
okay.
Thanks for the confirmation.
(1 liked)

Reply:
@sachinpunadikar please confirm that create interface alone is enough right now ?

Reply:
Thats correct @cdeshmukh . Currently we need create interface only